### DISCUSSION TITLE
Links in Markdown to HTML

### DISCUSSION THREAD

**Discussion by JaemeUK** [jgm/pandoc#10383] [General]
Hi,

I'm converting from .md to .html and the links are being strange.

If I convert ../characters/joe.md and there's a link in there to ../plan/decking.md then the link in joe.md comes out as ../characters/plan/decking.md 

Is there a way to set the root URL to '..' and the link ending to .html so the link comes out as ../plan/decking.html?

I hope that makes sense

**Comment by jgm** [comment]
Really?  That's not what I'm seeing (just tested).  Can you provide an example with all the files and commands necessary for testing, preferably in a zip so we don't need to create directories?
